CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ To resolve this:
 After making those changes, run `sudo systemctl daemon-reload`, and `sudo
 systemctl restart docker` to reload changes and (re)start the docker daemon.
 
+**IMPORTANT**: With Docker 1.12, a Linux docker installation now has two
+additional binaries; `dockerd`, and `docker-proxy`. If you have scripts for
```

Comment:
`dockerd` wasn't in 1.11 yet (yup, I also thought it was); https://github.com/docker/docker/pull/20639